main.py

```diff
@@ -1,11 +1,34 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
 
+    for player_name, player_info in players.items():
+        print(player_info)
+        race, _ = Race.objects.get_or_create(name=player_info["race"]["name"],
+                                   description=player_info["race"]["description"])
+
+
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(name=skill["name"],
+                                            bonus=skill["bonus"],
+                                            race=race)
+        if player_info["guild"]:
```

Comment:
Ensure that the `guild` variable is initialized to `None` before this conditional block. This will prevent a potential `UnboundLocalError` if `player_info['guild']` is `None` or missing.